### PR TITLE
Update deps and CI settings

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

None.

> Is there anything in the PR that needs further explanation?

Preparing for Stylelint 14.8.1 (see <https://github.com/stylelint/stylelint/issues/6043>)

### CODE REVIEW

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you.

Feel free to merge once stylelint is released and used here.

(Note: [deployments are manually triggered now](https://github.com/stylelint/stylelint-demo#deployment).)

**Review by jeddy3 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -152,13 +152,13 @@
...
-    "react": "^16.14.0",
+    "react": "^18.1.0",
     "react-css-modules": "^4.7.11",
-    "react-dom": "^16.14.0",
+    "react-dom": "^18.1.0",
     "rimraf": "^3.0.2",
     "serve-favicon": "^2.5.0",
     "style-loader": "^3.3.1",
-    "stylelint": "^14.8.0",
+    "stylelint": "github:stylelint/stylelint",
```
Comment (ybiquitous) [inline package.json:161]: TODO: Revert to `x.y.z`

**Review by jeddy3 — APPROVED** [review]

**Comment by ybiquitous** [comment]
I'm going to publish the demo site manually. 💪🏼

**Comment by ybiquitous** [comment]
Publishing done!

**Pull request merged**